versions.txt

```diff
@@ -0,0 +1,8 @@
+# this file contains the version of the Tempo components that will be used
+# by default with the Tempo Operator. This would usually be the latest
+# stable Tempo version. When you update this file, make sure to update the
+# the docs as well.
+tempo=1.5.0
```

Comment:
This info is as well defined in the https://github.com/os-observability/tempo-operator/blob/main/config/manager/controller_manager_config.yaml#L23

I would like to find an approach that minimizes duplications and makes it easy to bump versions. We should as well start a RELEASE doc and document how to bump versions for a release (might be a different PR).

Reply:
I think we don't need the tempo version there (`TEMPO_VERSION` is unused in the Makefile, because the version command now prints the default image).

About the operator version: can we use some git magic like `git describe --tags --always --dirty="-dev"` to generate the operator version?

Reply:
From the PR description:
> What is not included (but will be included in future PRs):
>
> * Change the Tempo version used by the operator
> * Possible needed changes in the bundle
> * Documentation

I'm following the same approach the Jaeger Operator and OpenTelemetry Operator are following. In the next PR I'll add the changes to modify the version of the used images and the documentation